mtp_api/apps/core/tests/test_dump_and_upload_for_ap.py

```diff
@@ -0,0 +1,32 @@
+from unittest import mock
+
+from django.core.management import call_command
+from django.test import TestCase, override_settings
+
+
+class TestDumpAndUploadForAp(TestCase):
+    @mock.patch('core.management.commands.dump_and_upload_for_ap.upload_dump_for_ap_command')
+    @mock.patch('core.management.commands.dump_and_upload_for_ap.dump_for_ap_command')
+    def test_dump_for_ap_command_is_called(self,
+                                           mock_dump_for_ap_command,
+                                           mock_upload_dump_for_ap_command):
+
+        call_command('dump_and_upload_for_ap')
+
+        assert mock_dump_for_ap_command.called
+
+    @mock.patch('core.management.commands.dump_and_upload_for_ap.upload_dump_for_ap_command')
+    @override_settings(ANALYTICAL_PLATFORM_BUCKET='my-bucket')
+    def test_upload_dump_for_ap_command_is_called(self, mock_upload_for_ap_command):
+
+        call_command('dump_and_upload_for_ap')
+
+        assert mock_upload_for_ap_command.called
+
+    @mock.patch('core.management.commands.upload_dump_for_ap.upload_to_s3')
+    @override_settings(ANALYTICAL_PLATFORM_BUCKET='my-bucket')
+    def test_upload_to_s3_is_called(self, mock_upload_to_s3):
+
+        call_command('dump_and_upload_for_ap')
+
+        assert mock_upload_to_s3.called
```

Comment:
when we were talking about not needing to know what `` was called with, that was specific to it since it will always be called with effectively random strings. for `upload_to_s3` we ought to check that the it's called twice with the expected bucket name and 2 different object names. that's pretty much the effect of this feature: deciding what to put into which bucket.